aspd/src/round/mod.rs

```diff
@@ -161,6 +161,10 @@ pub async fn run_round_scheduler(
 	let offboard_feerate = FeeRate::from_sat_per_vb(10).unwrap();
 	let round_tx_feerate = FeeRate::from_sat_per_vb(10).unwrap();
 
+	// The maximum number of output vtxos per round based on the max number
+	// of vtxo tree nonces we require users to provide.
+	let max_output_vtxos = (cfg.nb_round_nonces * 4 ) / 5;
```

Comment:
Because we expect users to pre-generate signing nonces when they sign up for a round, we need to tell them how many to provide. This limits the number of nodes in the tree and hence the number of leaves. If we exceed this, we would have to ask the user for more nonces somehow. We can put this high enough, or just have clients wait for the next round if one gets full.